src/sentry/static/sentry/app/views/projectInstall/overview.jsx

```diff
@@ -13,10 +13,11 @@ const ProjectInstallOverview = React.createClass({
 
   getIntegrationLink(root, platform, display) {
     let {orgId, projectId} = this.props.params;
+    let onboarding = this.props.location.query.onboarding ? 'onboarding=1' : '';
     return (
       <li className={`${root} ${platform}`} key={platform}>
         <span className={`platformicon platformicon-${platform}`}/>
-        <Link to={`/${orgId}/${projectId}/settings/install/${platform}/`}>
+        <Link to={`/${orgId}/${projectId}/settings/install/${platform}/?${onboarding}`}>
```

Comment:
I'd try and make sure we don't include the `?` either when `${onboarding}` is an empty string.